openapi/components/schemas/Organization.yaml

```
@@ -83,9 +83,119 @@ properties:
     pattern: '^[A-Z]{3}$'
     example: USD
   questionnaire:
-    $ref: ./Organizations/OrganizationQuestionnaire.yaml
+    type: object
+    nullable: true
+    description: Organization questionnaire.
+    properties:
+      role:
+        type: string
+        nullable: true
+        description: Role of the owner.
+      monthlyTransactions:
+        type: string
+        nullable: true
+        description: Amount of monthly processed transaction.
```

nitpick: I think this should be plural
```suggestion
        description: Amount of monthly processed transactions.
```